tests/js/spec/utils/withUserTeams.spec.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import {mount} from 'enzyme';
+
+import withUsersTeams from 'app/utils/withUsersTeams';
+
+describe('withUserTeams HoC', function() {
+  const api = new MockApiClient();
+  beforeEach(function() {
+    MockApiClient.clearMockResponses();
+  });
+
+  it('forwards errors', async function() {
+    MockApiClient.addMockResponse({
+      url: '/organizations/org-slug/user-teams/',
+      statusCode: 400,
+    });
+    const MyComponent = () => null;
+    const Container = withUsersTeams(MyComponent);
+    const wrapper = mount(<Container organization={{slug: 'org-slug'}} api={api} />);
```

Comment:
You can use `TestStubs.Organization()` to create an org fixture here (even though you only really need the slug)